### PR TITLE
Writing up of the contribution guide to the repository

### PR DESCRIPTION
**Pull request by mtendekuyokwa**
I have added starting points for people who might want to contribute. I took the installing guide for node and I added here. It felt more appropriete.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+### Getting Started
+
+This assumes that you have already set up your account information found in
+the [PawaPay API documentation](https://docs.pawapay.co.uk/).
+
+### Installation
+
+```bash
+npm install pawapay-node
```
Comment (joelfickson) [inline CONTRIBUTING.md]: Its supposed to be `npm install pawapay_api`

**Review by joelfickson — CHANGES_REQUESTED** [review]

**Comment by mtendekuyokwa** [comment]
I have made the update and even linted the doc. I hope thats Okay?

**Comment by joelfickson** [comment]
> I have made the update and even linted the doc. I hope thats Okay?

Yes this is good.

**Comment by mtendekuyokwa** [comment]
Thanks

**Pull request merged**